addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -386,16 +386,6 @@ return AbstractRenderer.extend({
         }
         this.trigger_up('openCreate', this._convertEventToFC3Event(data));
     },
-    /**
-     * Get the display time for an event in the same day but not all day
```

Comment:
I guess you fill fixup this into the previous. Do not forget to update the PR message accordingly. Thanks 🙂

Reply:
Indeed, I will remove the both commits. 🙂
Ok thanks, I will update it.